specification/application_layer/functions.tex

```diff
@@ -107,25 +108,26 @@ \subsection{Network-wide time synchronization}
 \subsection{Primitive types and physical quantities}
 
 The namespaces \DSDLReference{uavcan.si} and \DSDLReference{uavcan.primitive}
-included in the standard data type set are designed to provide a very generic and flexible,
-albeit neither bandwidth-efficient nor low-latency, method of real-time data exchange.
+included in the standard data type set are designed to provide a generic and flexible
+method of real-time data exchange; however, these namespaces are not bandwidth-efficient
```

Comment:
The properties of bandwidth utilization efficiency and latency are not defined on namespaces or data types contained therein. The new phrasing does not seem correct.